integration_test/testgenerated/auth/auth_test.go

```diff
@@ -62,7 +62,7 @@ func TestBothAuthClient(t *testing.T) {
 
 	// test invalid auth
 	_, err = client.Default(ctx, "invalid token")
-	assert.EqualError(t, err, "httpclient request failed: server returned a status >= 400")
+	assert.EqualError(t, err, "httpclient request failed: failed to unmarshal body as conjure error: json: cannot unmarshal string into Go value of type struct { Name string \"json:\\\"errorName\\\"\" }")
```

Comment:
not going to block on it, but something's fishy about this -- this either means we're returning JSON which doesn't match conjure or we're claiming it's application/json when it isn't

Reply:
Yeah, looks like https://github.com/palantir/witchcraft-go-server/blob/352f44636377416a16468535e8d8847e2137294d/rest/rest.go#L28 always writes that header.